docs/source/Quickstart.rst

```diff
@@ -314,3 +314,18 @@ Extracts a constraint from a cube. Time constraints work irrespective of whether
    >>> extracted_cube = ch.extract(cube, constraint)
    >>> extracted_cube
    <iris 'Cube' of air_temperature / (K) (time: 146; latitude: 145; longitude: 192)>
+
+Fixing known issues
+^^^^^^^^^^^^^^^^^^^
+There are some errors in CMIP data that are known about and can be fixed automatically. For details on the errors
+that can currently be fixed please see the :py:func:`cube_helper.fix_known_issues` documentation.
+
+.. code-block:: python
+
+   >>> cube = ch.load('/path/to/data/CMIP6/CMIP/CAS/FGOALS-f3-L/historical/r1i1p1f1/day/psl/gr/v20191019/')
+   >>> cube.coord('latitude').is_contiguous()
+   False
+   >>> ch.fix_known_issues(cube)
+   Applying FixCmip6CasFgoals
```

Comment:
Instead of `Applying FixCmip6CasFgoals` (which probably wouldn't mean anything to a typical user), would `Fixing latitude and longitude bounds` be more informative?

Reply:
I worry that `Fixing latitude and longitude bounds` isn't specific enough. By including the class name then users can refer to the documentation or source code for the exact details of what has been changed. In 362c2a8 and ec9cbbb I've added an additional message so that the class name can be included as well as a more informative message.